src/riot-rs/Cargo.toml

```diff
@@ -15,6 +15,7 @@ riot-rs-bench = { workspace = true, optional = true }
 riot-rs-boards = { path = "../riot-rs-boards" }
 riot-rs-debug = { workspace = true }
 riot-rs-embassy = { path = "../riot-rs-embassy" }
+riot-rs-embassy-common.workspace = true
```

Comment:
all occurrences of RIOT used to be C ;-)

More seriously: I think .workspace = true is an actual enhancement, which we should apply consistently (but this PR is not the place for it): Apart from the property form being generally less cluttered, the property form for .workspace is particularly well-suited because it won't cause edit churn with added properties (usually if you have .workspace you don't add other properties to it).

Reply:
Fine with me, applying. As with all this kind of style discussions, please consider whether these might be resolved more easily by enforcing them (then I care even less which direction they go).